### ISSUE TITLE
Multiple Selection

### ISSUE THREAD

**Issue by PabloReyes** [gfranko/jquery.selectBoxIt.js#81]
Is there any way to add support for multiple selection?
This guy (http://dingyonglaw.github.com/bootstrap-multiselect-dropdown/#forms) has already implemented it and it doesn't seem very difficult.

I'm trying to find some plugin that allows images and multiple selection, but the only one I found is Select2, which is great but too complex.

**Comment by apartamenti** [comment]
I agree we need multiple select.

**Comment by JackTheHack** [comment]
Hi. Is it feature implemented in v4.0?

**Comment by AlexMinaev** [comment]
+1 for multiple select option

**Comment by shr1611** [comment]
Hi @gfranko  ,
Is it implemented yet? Couldn't find any yet. If yes, can you share the link here. Thanks!

**Comment by JackTheHack** [comment]
@shr1611 I've made some changes to plugin that adds multiple select but it's a little bit messy in terms of code. If you are interested you can look at https://github.com/JackTheHack/jquery.selectBoxIt.js

**Comment by krystyna93** [comment]
Yes, can you please confirm when the multiselect function will be implemented??? 
Its 2017, I think its time to release it already..........zzzzzz

**Comment by maggenti** [comment]
Just to chime in with this; The problem with the default look of multiple is its can be really ugly! It can destroy a great looking design. Example would be a single horizontal line of dropdowns similar to a menu, then it jumps with this fat drop down. Deal breaker for me, man I was really excited until I investigated this part.  So back to Select2 I guess... but still looking?
